src/components/NoScriptWarning/style.scss

```diff
@@ -0,0 +1,13 @@
+.noscript_warning {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  z-index: 1201;
+  text-align: center;
+  font-weight: bold;
```

Comment:
remove bold, looks a bit odd.  and ensure we use the base font-size (14px)
looks more natural:
<img width="1425" alt="screen shot 2016-12-13 at 23 17 37" src="https://cloud.githubusercontent.com/assets/2606228/21163324/5f6900bc-c18a-11e6-9cdf-3db44a195c5f.png">